### ISSUE TITLE
Superheroes Unlimited and Transformers Mod Bug

### ISSUE THREAD

**Issue by Avenger1994** [Tihyo/SUM-Version#1]
Arm will not render for Transformer when the newest version(5.0) is installed

**Comment by Tihyo** [comment]
That's because both mods override player renders.

**Comment by Avenger1994** [comment]
Is there someway to fix the render thing, 'cause I like both the TF Mod Models and your suit's models

**Comment by Tihyo** [comment]
Not really. Try renaming the TF mod file and add a 1 in front of it to load
up first. That should fix SUMs animations.
